how-to/offline_cpu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Offline SOS CPUs except BSP before launch UOS
+
+for i in `ls -d /sys/devices/system/cpu/cpu[1-9]*`; do
+        online=`cat $i/online`
```

Comment:
Nit: I'd use `$( ... )` rather than backticks as they are clearer to read (here and below).

Reply:
done.